packages/smooth_app/lib/main.dart

```diff
@@ -69,6 +73,15 @@ Future<bool> _init1() async {
   if (_init1done) {
     return false;
   }
+
+  final PackageInfo packageInfo = await PackageInfo.fromPlatform();
+
+  OpenFoodAPIConfiguration.userAgent = UserAgent(
+    name: packageInfo.appName,
```

Comment:
This is:
> The app name. CFBundleDisplayName on iOS, application/label on Android.

But we can also just use Smoothie here as the app name can change due to translations or plain renaming @teolemon

Reply:
Smoothie + appname is better actually, given that we'll swap apps at some point.

Reply:
Like that?

Reply:
👍